test/e2e/login.test.ts

```diff
@@ -10,6 +10,12 @@ test.describe("login", () => {
     },
   }
 
+  test("should see the login page", options, async ({ page }) => {
+    await page.goto(CODE_SERVER_ADDRESS, { waitUntil: "networkidle" })
+    // It should send us to the login page
+    expect(await page.title()).toBe("code-server login")
+  })
+
```

Comment:
This was moved from `loginPage.test.ts`. I figured it made sense to put this here instead of having a separate file for that test.